airflow/dags/monitor/monitor.py

```diff
@@ -105,14 +117,52 @@ def check_firestore_status():
         firebase_admin.initialize_app()
         app = firebase_admin.get_app(name=firestore_app_name)
 
-        print(f"{app.name} found!")
+        logger.info(f"{app.name} found!")
+
+
+@task(trigger_rule=TriggerRule.ALL_DONE)
```

Comment:
Why does this need all_done? It has nothing upstream? copy/paste from slack_status?

Reply:
yes this should only be required for Slack task

Reply:
https://github.com/astronomer/ask-astro/pull/98/commits/c62341473efd7ad4ffb18de8255d23e73bd154eb